ajax/libs/bulmaswatch/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bulmaswatch",
+  "description": "Themes for Bulma",
+  "author": "Jenil Gogari <i@jgog.in>",
+  "homepage": "https://jenil.github.io/bulmaswatch",
+  "filename": "bulmaswatch.min.css",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jenil/bulmaswatch.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "css",
+    "sass",
+    "flexbox",
+    "responsive",
+    "framework",
+    "themes"
+  ],
+  "autoupdate": 
+  {
+    "source": "git",
+    "target": "git://github.com/jenil/bulmaswatch.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "bulmaswatch*.+(js|css|map)"
```

Comment:
@jenil then let's use the auto-update as the followings, this will get all the theme's files:
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/jenil/bulmaswatch.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "*/bulmaswatch*.+(js|css|map)"
        ]
      }
    ]
  }
```
And the `"filename": "bulmaswatch.min.css",` change to `"filename": "default/bulmaswatch.min.css",` let it as the main file.
Thank you.